src/builtins/hmr-runtime.js

```diff
@@ -22,6 +22,10 @@ if (!module.bundle.parent) {
     if (data.type === 'update') {
       for (let asset of data.assets) {
         hmrApply(global.require, asset);
+        // CSS should always update
+        if (asset.generated && asset.generated.css) {
+          asset.isNew = false;
+        }
```

Comment:
Not sure I understand why this was added. Can you explain?

Reply:
First time css gets edited it doesn't have it inside modules so it only updates from second edit up.
This should fix that, there is potentially a better fix for it though but thought this fix wasn't that bad because every generated js for css updates is the same anyway.

Reply:
Ah interesting. That's an interesting issue. Seems that the HMR code is not actually getting bundled into the JS output. Will look into it a bit more.